bench/workgen/wtperf.py

```diff
@@ -467,9 +481,10 @@ def translate_inner(self):
                 if len(line) == 0:
                     continue
                 (key, val) = self.split_assign(line)
+                # Note: 'sample_file' is not an original wtperf option.
                 if key in [ 'max_latency', 'report_file', 'report_interval',
-                            'run_time', 'sample_interval', 'sample_rate',
-                            'warmup' ]:
+                            'run_time', 'sample_file', 'sample_interval',
```

Comment:
sample_stat_file or stat_file might be a more meaningful name for this parameter.

Reply:
Removed this new option, see previous comment.